### DISCUSSION TITLE
Library Panels: Add create new panel button to the library panel browser

### DISCUSSION THREAD

**Discussion by tskarhed** [grafana/grafana#38314] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
A button for creating a new library panel in the library panel browsing view.
 
![Screenshot from 2021-06-28 10-46-44](https://user-images.githubusercontent.com/1438972/123607742-720c3600-d7fe-11eb-8f61-cea1438270ff.png)

**Why is this needed**:

It feels like a natural approach. I didn't realize how "deep" you have to go in order to create one.

**Comment by torkelo** [comment]
@tskarhed problem is how would it work? we do not have a standalone panel edit UI. The button would need to create and save a new dashboard.

**Comment by tskarhed** [comment]
Yeah, I've realized now that the LibraryPanels aren't decoupled from the dashboards. It just feels like the expected behaviour for a user.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
